backend/queries/queries.go

```diff
@@ -5,100 +5,140 @@ import (
+					TotalCount graphql.Int
+				} `graphql:"commits(first: 1)"`
+				Repository struct {
+					Url             graphql.String
+					Name            graphql.String
+					PrimaryLanguage struct {
+						Name graphql.String
+					}
+				}
 			}
-		} `graphql:"pullRequests(first: 50, states:MERGED)"`
-	} `graphql:"user(login:$username)"`
+		} `graphql:"pullRequests(first:40, states: MERGED)"`
+	} `graphql:"user(login: $username)"`
 }
 
-type commitsOnPRs struct {
+type InvolvedIssues struct {
 	User struct {
-		PullRequests struct {
+		IssueComments struct {
 			TotalCount graphql.Int
 			Nodes      []struct {
-				Url       graphql.String
-				CreatedAt graphql.String
-				Commit    struct {
-					TotalCount graphql.Int
-				} `graphql:"commits(first: 1)"`
+				Issue struct {
+					Title graphql.String
+				}
+				Url        graphql.String
+				Repository struct {
+					Name            graphql.String
+					Url             graphql.String
+					PrimaryLanguage struct {
+						Name graphql.String
+					}
+				}
 			}
-		} `graphql:"pullRequests(first: 30, states:MERGED)"`
-	} `graphql:"user(login:$username)"`
+		} `graphql:"issueComments(first: 40)"`
+	} `graphql:"user(login: $username)"`
+}
+
+type IssuesOpenVsClosed struct {
+	User struct {
+		Issues struct {
+			TotalCount graphql.Int
+			Nodes      []struct {
+				State graphql.String
+			}
+		} `graphql:"issues(first: 30, states:[OPEN,CLOSED])"`
+	} `graphql:"user(login: $username)"`
 }
 
-type repositoriesContributedTo struct {
+type ReposContributedTo struct {
 	User struct {
 		PullRequests struct {
 			TotalCount graphql.Int
 			Nodes      []struct {
-				CreatedAt       graphql.String
 				Name            graphql.String
+				Description     graphql.String
 				Url             graphql.String
 				PrimaryLanguage struct {
 					Name graphql.String
 				}
-				Languages struct {
-					Nodes []struct {
-						Name graphql.String
-					}
-				} `graphql:"languages(first: 5)"`
+				ForkCount  graphql.Int
+				StarGazers struct {
+					TotalCount graphql.Int
+				} `graphql:"stargazers(first: 1)"`
 			}
-		} `graphql:"repositoriesContributedTo(first: 25, contributionTypes:[PULL_REQUEST])"`
+		} `graphql:"repositoriesContributedTo(first: 25)"`
 	} `graphql:"user(login:$username)"`
 }
 
-type pullRequests struct {
+type PRsMergedVsNot struct {
 	User struct {
 		PullRequests struct {
-			Nodes []struct {
-				CreatedAt graphql.String
-				Merged    graphql.Boolean
-				Url       graphql.String
-			}
-		} `graphql:"pullRequests(first:30)"`
-	} `graphql:"user(login: $username)"`
-}
-
-type issuesCreated struct {
-	User struct {
-		Issues struct {
 			TotalCount graphql.Int
 			Nodes      []struct {
-				Url       graphql.String
-				CreatedAt graphql.String
-				Closed    graphql.Boolean
+				Merged graphql.Boolean
 			}
-		} `graphql:"issues(first:20)"`
-	} `graphql:"user(login: $username)"`
+		} `graphql:"pullRequests(first: 50, states:[OPEN,CLOSED,MERGED])"`
+	} `graphql:"user(login:$username)"`
 }
```

Comment:
Could this be done like I suggested above?
```graphql
User {
   PullRequests {
      totalcount
   } states:[OPEN]
   PullRequests {
      totalcount
   } states:[CLOSED]
   PullRequests {
      totalcount
   } states:[MERGED]
} 
```

Reply:
For this query I was specifically looking to implement to this merged vs non merged PR count.

![this](https://cdn.discordapp.com/attachments/735957303768907816/735965316001038356/23_07_20_215826_.png)

A new query to get the stats in your comment might be possible. I'm not against it actually I think it'd be better to have a graph with those 3 metrics compared to the way we had it before. But maybe showing closed PRs isn't the best as that's not really an achievement if they're not merged